types/vvvv.shared_types.js

```diff
@@ -72,4 +72,20 @@ VVVV.Types.CanvasGraphics = {
   
 }
 
+VVVV.Types.Buffer = function(vectorSize, data, offset, count, id) {
+  this.VectorSize = vectorSize;
+  this.data = data;
+  this.offset = offset;
+  this.count = count;
+  this.id = id;
+  }
+
+VVVV.PinTypes.Buffer = {
```

Comment:
This will clash with another pin type in the 'require' branch, which is also called `Buffer`.  It is uses for standard JavaScript raw byte data (similar to VVVV's `Raw` nodes)

Maybe we can rename this to something more WebGL related (without calling it WebGlBuffer) ?